health-checker/src/main/java/io/stargate/health/HealthCheckerActivator.java

```diff
@@ -22,13 +26,22 @@ public HealthCheckerActivator() {
   }
 
   @Override
-  public synchronized void stop(BundleContext context) {}
+  public synchronized void stop(BundleContext context) {
+    healthCheckRegistry.get().unregister(BUNDLES_CHECK_NAME);
+    healthCheckRegistry.get().unregister(DATASTORE_CHECK_NAME);
+  }
 
   @Nullable
   @Override
   protected ServiceAndProperties createService() {
     log.info("Starting healthchecker....");
     try {
+      BundleStateChecker bundleStateChecker = new BundleStateChecker(context);
+      healthCheckRegistry.get().register(BUNDLES_CHECK_NAME, bundleStateChecker);
+
+      DataStoreHealthChecker dataStoreHealthChecker = new DataStoreHealthChecker(context);
+      healthCheckRegistry.get().register(DATASTORE_CHECK_NAME, dataStoreHealthChecker);
```

Comment:
why don't treat the `DATASTORE_CHECK_NAME` as a normal `BUNDLES_CHECK_NAME`? If we would do that, we won't need this special case. It would be handled by the `BundleStateChecker`

Reply:
I assumed we wanted to delineate checks by their function to make trouble-shooting easier. Different names will ensure that we expose information about these checks in separate JSON elements under `/healthcheck`.